### PR TITLE
Feature/frontend/hmi finalizing layout, Finalized HMI Version Without YAAC Notfications

### PR DESCRIPTION
**Pull request by danielSiegt08**

### CODE REVIEW

**Review by MaxAlberti — CHANGES_REQUESTED** [review]
Crashes right after start:
```log
2024/03/23 15:23:33 Info: dial tcp: missing address
2024/03/23 15:23:33 Trying connect to mail server without TLS ...
2024/03/23 15:23:33 Error: dial tcp: missing address
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0xba6237]

goroutine 66 [running]:
github.com/DHBW-SE-2023/YAAC/internal/backend/mail.(*BackendMail).setupMail(0x0, 0x1)
        /run/host/home/max/Code/YAAC/internal/backend/mail/mailClient.go:243 +0x17
github.com/DHBW-SE-2023/YAAC/internal/backend/mail.(*BackendMail).GetMailsToday(0x0)
        /run/host/home/max/Code/YAAC/internal/backend/mail/mailClient.go:29 +0x25
github.com/DHBW-SE-2023/YAAC/internal/backend/demon.StartDemon({0xfe3148, 0xc0002d31e0}, 0xa)
        /run/host/home/max/Code/YAAC/internal/backend/demon/demon.go:15 +0x50
created by github.com/DHBW-SE-2023/YAAC/internal/mvvm.(*MVVM).StartDemon in goroutine 1
        /run/host/home/max/Code/YAAC/internal/mvvm/demon.go:12 +0x69
make: *** [Makefile:26: run] Error 2
```
I assume the demon is not initialized correctly/nil.

File: assets/alive.png
Comment (MaxAlberti) [inline assets/alive.png:1]: Please bundle all resources!!!

File: assets/down.png
Comment (MaxAlberti) [inline assets/down.png:1]: Please bundle all resources!!!

File: assets/imageIcon.png
Comment (MaxAlberti) [inline assets/imageIcon.png:1]: Please bundle all resources!!!

File: assets/ImageUpload.png
Comment (MaxAlberti) [inline assets/ImageUpload.png:1]: Please bundle all resources!!!

File: assets/.DS_Store
Comment (MaxAlberti) [inline assets/.DS_Store:1]: Delete this?

**Review by MaxAlberti — CHANGES_REQUESTED** [review]
Please fix

File: dataImport.sh
Comment (MaxAlberti) [inline dataImport.sh:1]: Error:
```log
↪  ./dataImport.sh                                                                                                Sat Mar 23 15:26:38 CET 2024
Inserting Values
Parse error near line 4: no such column: NULL
  Update Course set DeletedAt=NULL where DeletedAt="NULL";
                                     error here ---^
```

**Review by MaxAlberti — CHANGES_REQUESTED** [review]
Error with script

File: settings.csv
Comment (MaxAlberti) [inline settings.csv:1]: Encryption? This does not seem compliant. Just temporary?

**Review by MaxAlberti — CHANGES_REQUESTED** [review]
Unprotected settings?

File: assets/alive.png
Comment (danielSiegt08) [reply assets/alive.png:1]: All resources got bundled

File: assets/down.png
Comment (danielSiegt08) [reply assets/down.png:1]: All resources got bundled

File: assets/imageIcon.png
Comment (danielSiegt08) [reply assets/imageIcon.png:1]: All resources got bundled

File: assets/ImageUpload.png
Comment (danielSiegt08) [reply assets/ImageUpload.png:1]: All resources got bundled

File: dataImport.sh
Comment (danielSiegt08) [reply dataImport.sh:1]: The script won't be used in the future since we'll provide a fully configured database.

**Review by MaxAlberti — APPROVED** [review]

**Pull request merged**